lib/components/molecules/Accordion/AccordionPanel/AccordionPanel.story.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+// import { action } from '@storybook/addon-actions';
+import defaultConfig from './AccordionPanel.mock';
+
+// Import Styled Component to showcase variations
+import AccordionPanel, { AccordionPanelVanilla } from '.';
+
+storiesOf('Molecules/AccordionPanel', module)
```

Comment:
There is no need to have storybook config for just header and panel, we can't test these independently.